go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/project-copacetic/copacetic
 
-go 1.22.0
+go 1.23.0
```

Comment:
afaik this signals that 1.23 is the min version require to import this. copa isn't designed to be use as a library and not sure why would we want 1.23 as a min version?